dashboard/legacy/middleware/animation_library_api.rb

```diff
@@ -134,22 +133,6 @@ class AnimationLibraryApi < Sinatra::Base
     not_found
   end
 
-  #
-  # GET /api/v1/animation-library/default-spritelab/
-  #
-  # Retrieve the default sprite list from S3
-  get %r{/api/v1/animation-library/default-spritelab} do
```

Comment:
The same thing that happened in AnimationLIbraryApi.js is happening here - removing duplicated functions.